src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    public String getRandomColor() {
```

Comment:
The method `getRandomColor` is declared to return a `String`, but it should return a `Color` since you are selecting from `Color.values()`. Change the return type of the method to `Color`.